src/pcacv/bin/run_pca_cv.R

```diff
@@ -273,6 +274,12 @@ RunPCACV <- function(
 			object = object
 			, path = args$`accessor`
 		)
+		meta_features <- object@assays$RNA@meta.features
+		# FIXME: what do we do when there are more than 1 variable feature column in the data 
+		# e.g. after running SCT and FindVariableFeatures on the same dataset. 
+		# This is highly unlikely, but we should still catch it here.
```

Comment:
I don't know if taking the union as a good idea. Sometimes the HVG between SCT and FindVariableFeatures can be quite different. When you then take the union, you risk getting only 1000 genes. This can be on the lower side (depending on the complexity of the data).
I will just use the HVG from the current default assay instead of always defaulting to the RNA assay. In case SCT is used, the default assay in Seurat should automatically be set to SCT.

Reply:
I tried adding what I mentioned above, but ran into some issues with the Seurat version in the pca_cv container.
It needs a version >= 4.0.0. I don't want to mess too much with this container and risk breaking compatibility with other parts of the pipeline, so not sure how to proceed

Reply:
@kverstae I've built a new `pcacv` container which contains R version 4.0.0: `vibsinglecellnf/pcacv:0.3.0`. Sorry for the late response.

Reply:
@dweemx As far as I can see it still contains an older version of Seurat (3.1.5), but I need something >= 4.0.0
Not everything of Seurat 4.0.x is backwards compatible with the older Seurat 3.x.x versions.